cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
I am confused. Is this not splitting the host folder config into two places now? AFAICS, there is still HostFolders in InstanceConfig. How is this going to work?

It seems you keep whether automounting occurs in this config as well as the mount path!? So what if I delete the VM and create a new one? Now I would have to first re-configure the basic host folder configuration, before auto mounting would work again, right?